SignalServiceKit/protobuf/OWSSignalServiceProtos.proto

```diff
@@ -238,3 +238,21 @@ message GroupDetails {
   optional bool   active      = 5 [default = true];
   optional uint32 expireTimer = 6;
 }
+
+message BackupSnapshot
```

Comment:
Since we're putting this work on pause, I've temporarily put these classes in the "signal service" proto schema, but ultimately they should be pulled out into an "signal ios" proto schema.

Reply:
No biggie - but if we try to pull in upstream changes on this file, its now going to be a bit more of a pain to merge them - is there a reason to not put them in their final resting place now?

Reply:
Done.